lib/dolly/property.rb

```diff
@@ -4,7 +4,7 @@ class Property
     attr_accessor :name
     attr_reader :class_name, :default
 
-    CANT_CLONE = [NilClass, TrueClass, FalseClass, Fixnum].freeze
+    CANT_CLONE = [NilClass, TrueClass, FalseClass, Integer].freeze
```

Comment:
I think this, in order to be useful in a lot of places, should be determined based on the version of ruby (as I think that is the change in this no?)

Reply:
mmh, right, this is not related to the ruby version
will remove the change as it is not related to the errors associated to httparty parsed response

Reply:
👍 ya i think a PR for that on its own makes sense. I could see a singleton of a DelegateClass Array here as a feasible way to handle that